hooks/usePoolInstanceActions/index.ts

```diff
@@ -89,7 +92,7 @@ export const usePoolInstanceActions = (): PoolInstanceActions => {
                     onSuccess: (receipt) => {
                         console.debug('Successfully submitted claim txn: ', receipt);
                         // get and set token balances
-                        updateTokenBalances(pool, provider, account);
+                        updatePoolTokenBalances([pool], provider, account);
```

Comment:
I think we want to update settlement token balances after a claim too since settlement token could be moving from escrow to wallet